documentation_crawl4ai/async-webcrawler.md

```
@@ -0,0 +1,386 @@
+from crawl4ai import CrawlerRunConfig, CacheMode
+
+run_cfg = CrawlerRunConfig(
+    cache_mode=CacheMode.BYPASS,
+    css_selector="main.article",
+    word_count_threshold=10,
+    screenshot=True
+)
+
+async with AsyncWebCrawler(config=browser_cfg) as crawler:
+    result = await crawler.arun("https://example.com/news", config=run_cfg)
+    print("Crawled HTML length:", len(result.cleaned_html))
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)